src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,25 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/** This abstract class is a generic class which all other command classes inherit from. */
+public abstract class Command {
+    protected boolean exit;
```

Comment:
I think the naming for `exit` variable could be clearer and sound more like a boolean, so maybe you could consider `isExit`? Just a suggestion!